dvc/external_repo.py

```diff
@@ -219,8 +220,10 @@ def _clone_default_branch(url, rev):
             git = Git(clone_path)
             # Do not pull for known shas, branches and tags might move
             if not Git.is_sha(rev) or not git.has_rev(rev):
+                logger.debug("erepo: git pull {}", url, rev)
```

Comment:
I think, we need two braces here:
```suggestion
                logger.debug("erepo: git pull {} {}", url, rev)
```

Reply:
We don't need `rev` here.